### PR TITLE
ci(github-actions): add workflow to deploy integration tests to astro cloud

### PR DESCRIPTION
**Pull request by Lee-W**
as title

### CODE REVIEW

File: .github/workflows/deploy-integration-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        description: 'astronomer_key_id'
+        required: true
+        default: ''
+      astronomer_key_secret:
+        description: 'astronomer_key_secret'
+        required: true
+        default: ''
+
+jobs:
+  deploy-integeration-tests-to-cloud:
```
Comment (phanikumv) [inline .github/workflows/deploy-integration-tests.yaml:33]:
```suggestion
  deploy-integration-tests-to-astro-cloud:
```

File: .github/workflows/deploy-integration-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+name: deploy-integeration-tests-to-cloud
```
Comment (phanikumv) [inline .github/workflows/deploy-integration-tests.yaml:2]:
```suggestion
name: deploy-integration-tests-to-astro-cloud
```

File: .github/workflows/deploy-integration-tests.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+jobs:
+  deploy-integeration-tests-to-cloud:
+    runs-on: 'ubuntu-20.04'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          ref: '${{ github.event.inputs.git_rev }}'
+
+      - name: delploy
```
Comment (pankajkoti) [inline .github/workflows/deploy-integration-tests.yaml:41]:
```suggestion
      - name: deploy
```

**Review by phanikumv — COMMENTED** [review]
can you deploy current main branch on https://cloud.astronomer.io/cldmz5pu62139431i04uatk0poa/deployments/clgw5tzfs84818b7vvj9rsqnhs/analytics after merging this?

File: .github/workflows/deploy-integration-tests.yaml (hunk 1 above)
Comment (Lee-W) [reply .github/workflows/deploy-integration-tests.yaml]: Fixed. Thanks!

File: .github/workflows/deploy-integration-tests.yaml (hunk 3 above)
Comment (Lee-W) [reply .github/workflows/deploy-integration-tests.yaml]: Fixed. Thanks!

File: .github/workflows/deploy-integration-tests.yaml (hunk 2 above)
Comment (Lee-W) [reply .github/workflows/deploy-integration-tests.yaml]: Fixed. Thanks!

**Comment by Lee-W** [comment]
> can you deploy current main branch on https://cloud.astronomer.io/cldmz5pu62139431i04uatk0poa/deployments/clgw5tzfs84818b7vvj9rsqnhs/analytics after merging this?

Sure!

**Comment by Lee-W** [comment]
As also the comments are addressed and is now approved, I'll merge this one.

**Pull request merged**